src/py/run_generator.rs

```diff
@@ -0,0 +1,290 @@
+use crate::allocator::{Allocator, NodePtr};
+use crate::cost::Cost;
+use crate::gen::conditions::{parse_spends, Condition, SpendConditionSummary};
+use crate::gen::opcodes::{
+    ConditionOpcode, AGG_SIG_ME, AGG_SIG_UNSAFE, ASSERT_HEIGHT_ABSOLUTE, ASSERT_HEIGHT_RELATIVE,
+    ASSERT_SECONDS_ABSOLUTE, ASSERT_SECONDS_RELATIVE, CREATE_COIN, RESERVE_FEE,
+};
+use crate::gen::validation_error::{ErrorCode, ValidationErr};
+use crate::int_to_bytes::u64_to_bytes;
+use crate::py::run_program::OperatorHandlerWithMode;
+use crate::reduction::{EvalErr, Reduction};
+use crate::run_program::{run_program, STRICT_MODE};
+use crate::serialize::node_from_bytes;
+
+use crate::py::f_table::f_lookup_for_hashmap;
+use crate::py::lazy_node::LazyNode;
+
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use pyo3::prelude::*;
+use pyo3::types::{PyBytes, PyDict};
+
+#[derive(Clone)]
+pub struct PyBytesObj {
+    arr: Vec<u8>,
+}
+
+impl PyBytesObj {
+    pub fn from_vec(v: Vec<u8>) -> PyBytesObj {
+        PyBytesObj { arr: v }
+    }
+
+    pub fn from_node(a: &Allocator, n: NodePtr) -> PyBytesObj {
+        PyBytesObj {
+            arr: a.atom(n).to_vec(),
+        }
+    }
+
+    pub fn from_int(n: u64) -> PyBytesObj {
+        let buf = u64_to_bytes(n);
+        PyBytesObj { arr: buf.to_vec() }
+    }
+}
+
+impl IntoPy<PyObject> for PyBytesObj {
+    fn into_py(self, py: Python) -> PyObject {
+        PyBytes::new(py, &self.arr).into()
+    }
+}
+
+#[pyclass(subclass, unsendable)]
```

Comment:
I wonder if these go away if we upgrade `pyo3` to 0.14.1. Might be worth trying.

Reply:
I'm trying it. Use `cargo +nightly clippy` to run nightly clippy locally (might have to do something like `rustup update nightly` if you haven't yet).

Reply:
Yep, that fixes it!